pkg/kubectl/describe.go

```diff
@@ -382,13 +382,13 @@ func (d *ResourceQuotaDescriber) Describe(namespace, name string, describerSetti
 func helpTextForResourceQuotaScope(scope api.ResourceQuotaScope) string {
 	switch scope {
 	case api.ResourceQuotaScopeTerminating:
-		return "Matches all pods that have an active deadline."
+		return "Affects pods that have an active deadline. These pods usually include builds, deployers, and jobs."
 	case api.ResourceQuotaScopeNotTerminating:
-		return "Matches all pods that do not have an active deadline."
+		return "Affects pods that do not have an active deadline. These pods usually include those which host an application."
```

Comment:
"host an application", is really vague.  Also, this is specifically for matching, why are you switching to "Affects"?